src/main/archetype/pom.xml

```diff
@@ -154,15 +154,6 @@
                     <artifactId>maven-install-plugin</artifactId>
                     <version>2.5.2</version>
                 </plugin>
-                <!-- Maven Surefire Plugin -->
```

Comment:
I chose to leave the Surefire plugin in, I just upgraded it and the Failsafe plugin to 2.22.1.  These versions are compatible with JUnit5.